packages/model/src/working-group.ts

```diff
@@ -39,6 +40,7 @@ export type WorkingGroupMember = {
     | 'email'
     | 'avatarUrl'
   >;
+  readonly isActive: boolean;
```

Comment:
by adding `isActive` property to the `WorkingGroupMember` you are effectively adding it to `WorkingGroupDataObject` parent object which I think is not a valid approach. The reason is that our data-providers should only be an abstraction layer on top of the data store (CMS) which send the data back and forth and they should be fairly 'dumb' in a sense that they should not really perform any complicated logic and should not return properties calculated 'on the go'.

Although `WorkingGroupResponse` is currently the same as `WorkingGroupDataObject` I think you should make sure that the new property is added only to members on the `WorkingGroupResponse` type. This means you are going to have to perform the check in the controller but that's ok.